starter-code/routes/drones.js

```diff
@@ -1,12 +1,21 @@
 const express = require('express');
-
+const Drones = require('../models/Drones').Drone;
 // require the Drone model here
 
 const router = express.Router();
 
 
 router.get('/drones', (req, res, next) => {
-  // Iteration #2
+  Drones.find({}, (error, result) => {
+    if (error) {
+      next(error);
+    } else {
+      const data = {
+        drones: result
+      }
+      res.render('drones', data);
```

Comment:
`res.render('drones', data);`
drones here is the ejs file in the view folder. I recommend, create a folder for drones and create for example a file called: ListDrones.ejs. To the path would be "drones/ListDrones"